HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -1872,113 +1872,120 @@ def getPostStates(self):
         self.aLvlNow = self.aNrmNow*self.pLvlNow   # Useful in some cases to precalculate asset level
         return None
 
-    def checkAIC(self, thorn,verbose,public_call):
+    def checkCondition(self,
+                       name,
+                       test,
+                       messages,
+                       verbose=False):
+        """
+        Checks one condition.
+
+        Parameters
+        ----------
+        name : string
+             Name for the condition.
+
+        test : function(self -> boolean)
+             A function (of self) which tests the condition
+
+        messges : tuple[string]
+            A length four tuple of string to be printed to the 
+            user, in order: True, always; True, verbose; 
+            False, always; False, verbose.
+
+        verbose : boolean
+            (Optional) A boolean to set verbose printing
+        """
+        self.conditions[name] = test(self)
+
+        print(messages[0].format(self)
```

Comment:
Give names to messages 0, 1, 2, like 

condTru=0
condFalse=1

so then you could have print(messages[condTru]) which would be more transparent